packages/tools/fuzzer/model/ActionTracker.ts

```
@@ -18,11 +21,80 @@ interface ActionLogEntry {
+		['string', { email: 'string', childIds: 'string[]' }],
+		'...',
+	],
+} satisfies BaseSchema;
+
+class ActionTracker extends Serializable<typeof schema> {
 	private idToActionLog_: Map<ItemId, ActionLogEntry[]> = new Map();
 	private idToItem_: Map<ItemId, TreeItem> = new Map();
 	private tree_: Map<string, ClientData> = new Map();
-	public constructor(private readonly context_: FuzzContext) {}
+
+	public constructor(private readonly context_: FuzzContext) {
```

ESLint

[error] 'context_' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)